.github/workflows/push_docker.yaml

```diff
@@ -18,11 +18,20 @@ on:
       - 'LICENSE'
       - 'setup.py'
       - 'pyproject.toml'
-
+  pull_request:
+    branches:
+      - multi-arch-docker-build
 jobs:
   build_and_push:
     if: github.repository_owner == 'ml-energy'
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
What is this one?

Reply:
It ensures that both platform builds will run to completion, regardless of whether one of them fails. Do you think this is unnecessary?

Reply:
Is fail-fast `true` by default? I think we actually want to fail fast. If one platform failed, it means something is wrong with the Dockerfile and we don't want to upload a partial image.

Reply:
I have set fail-fast value to true as it is not true by default.

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast I think it's true by default. If it is so, let's delete the line.

Reply:
Ah sorry, it seems to be true by default. Removed the line!